### DISCUSSION TITLE
Default Preview Image

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2758] [Ideas]
Preview images are pretty important for layouts that use them, but often they're an afterthought for submitters. It would be nice to be able to have an alternative to the random color gradient tiles that currently serve as preview images, either based off of community colors, integrated into cascading settings, or to default to using the header image unless otherwise specified.

**Comment by kyledenlinger** [comment]
Wanted to ping this one again. I have some large(ish) communities with lots of pubs that lack preview images. I'd love to replace the default with a community logo or something like that. It's possible to replace the header background for all pubs either at the community level or at the collection level; it'd be wonderful to have the same control over preview images.

**Comment by tefkah** [comment]
I agree, makes sense to have some more control over it globally, and I like the idea of the preview image to be the header background by default! There's already a button to set it to that, why does you need to click it every time?

**Comment by amdeis** [comment]
Might be helpful to have a "preview image" option for collection-wide pub settings, like exists for the "background image"

**Comment by meeloun** [comment]
When choosing Canadian proxy examination        [https://www.lunwentop.net/jia-na-da-dai-kao/](https://www.lunwentop.net/jia-na-da-dai-kao/)       services, it is important to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.
